focus-areas/people/contributors.md

```diff
@@ -1,75 +1,98 @@
+# **Contributors**
+
+**Question:** Who are the contributors to a project?
+
+## **Overview**
+A contributor is defined as anyone who contributes to the project in any way. This metric ensures that all types of contributions are fully recognized within the project. Identifying contributors helps the community recognize and celebrate various contributions, from code development to event planning and marketing efforts.
+
+Recognizing the contributors is vital to understanding who is driving different activities and informs project health by showing diversity and breadth in project involvement.
+
+## **Want to Know More?**
+
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
+
+### **Data Collection Strategies**
+- **Aggregators:** Collect contributor names from collaboration tools, such as source code repositories, issue trackers, event registrations, mailing lists or any other tools a project uses.
+   - *Count:* Total number of contributors during a given time period.
+- **Parameters:**
+   - *Period of time:* Start and end dates for the period in which contributions are counted (default: forever).
+- **Data Sources:** Contributor data can be collected from collaboration platforms like GitHub, GitLab, IRC, blogs, and forums, or from tools like GrimoireLab and Augur.
+
+### **Filters**
+Contribution data can be filtered in several ways, such as:
+- By the location of engagement:
+   - Commit authors
+   - Issue authors
+   - Review participants (e.g., pull requests)
+   - Mailing list authors
+   - Event participants
+   - IRC or forum authors
+- By project characteristics:
+   - By release cycle
+   - Programming languages of the project
+   - Role or function in the project (e.g., coding, organizing, mentoring)
+   - Timeframe of activity (e.g., identifying new contributors)
+
+### **Visualizations**
+1. **List of Contributors:**  
+   Display contributor names with engagement details.
+<img width="490" alt="IMG_5268" src="https://github.com/user-attachments/assets/456864b5-6632-4fda-93a6-0ada9ca10b45">
+ 
+  *Figure 1: List of contributors and engagement (Source, Year)*
+
+2. **Summary of Contributors:**  
+   A summary showing the total number of contributors.
+![IMG_5269](https://github.com/user-attachments/assets/1d60591c-3024-4068-82ed-241846c03ac7)
+
+   *Figure 2: Summary number of contributors (Source, Year)*
+
+4. **Growth Over Time:**  
+   Change in the number of active contributors over time.
+![IMG_5270](https://github.com/user-attachments/assets/c13de6dc-8f08-4d5d-b98b-dba0b3673dc8)
+
+    *Figure 3: Contributor growth over time (Source, Year)*
```

Comment:
Could you please remove 'Year' from (Source, Year). 

Could you please change 'Source' to the name of the software that provided the metric. If you are not sure, please leave blank.

Reply:
Done, Thank you Matt. I did leave it blank because I wasn’t sure. Is there a way to know which is for GrimoireLab or Augur?